diff --git a/onnx/defs/schema.h b/onnx/defs/schema.h
--- a/onnx/defs/schema.h
+++ b/onnx/defs/schema.h
@@ -1004,20 +1004,20 @@ class OpSchemaRegisterOnce {
     OpSchemaRegisterOnce( // NOSONAR
         OpSchema op_schema,
         int opset_version_to_load = 0,
-        bool fail_duplicate_schema = true) {
+        bool fail_duplicate_schema = false) {
       OpSchemaRegisterNoExcept(std::move(op_schema), opset_version_to_load, fail_duplicate_schema);
     }
     ONNX_API static void
-    OpSchemaRegisterNoExcept(OpSchema&& op_schema, int opset_version_to_load = 0, bool fail_duplicate_schema = true) {
+    OpSchemaRegisterNoExcept(OpSchema&& op_schema, int opset_version_to_load = 0, bool fail_duplicate_schema = false) {
       ONNX_TRY {
         OpSchemaRegisterImpl(std::move(op_schema), opset_version_to_load, fail_duplicate_schema);
       }
       ONNX_CATCH(const std::exception& e) {
         ONNX_HANDLE_EXCEPTION([&]() { std::cerr << "Schema error: " << e.what() << '\n'; });
       }
     }
     ONNX_API static void
-    OpSchemaRegisterImpl(OpSchema&& op_schema, int opset_version_to_load = 0, bool fail_duplicate_schema = true) {
+    OpSchemaRegisterImpl(OpSchema&& op_schema, int opset_version_to_load = 0, bool fail_duplicate_schema = false) {
       op_schema.Finalize();
       auto& m = GetMapWithoutEnsuringRegistration();
       auto& op_name = op_schema.Name();
